pkg/render/compliance.go

```diff
@@ -1021,6 +1027,10 @@ func (c *complianceComponent) complianceSnapshotterDeployment() *appsv1.Deployme
 				MountPath: LinseedVolumeMountPath,
 			})
 	}
+	var initContainers []corev1.Container
```

Comment:
How does this work for compliance reporter ?

Reply:
Added, thanks